src/BaGet.UI/src/DisplayPackage/Dependents.tsx

```diff
@@ -49,19 +78,37 @@ class Dependents extends React.Component<IDependentsProps, IDependentsState> {
 
     if (this.state.totalHits === 0) {
       return (
-        <div>No packages depend on {this.props.packageId}.</div>
+      <div>{this.getDependentsMessage()}</div>
       );
     }
 
     return (
         <div>
-          <p>{this.state.totalHits} {this.state.totalHits === 1 ? 'package depends' : 'packages depend' } on {this.props.packageId}:</p>
+          <p>{this.getDependentsMessage()}</p>
           <div>
-            <ul>
-              {this.state.data.map(dependent => (
-                <li key={dependent}>{dependent}</li>
-              ))}
-            </ul>
+            <table>
+              <thead>
+                <tr>
+                  <th className="col-sm-10">Package</th>
+                  <th className="col-sm-2">Downloads</th>
+                </tr>
+              </thead>
+              <tbody>
+                {this.state.data.map(dependent => (
+                  <tr key={dependent.id}>
+                    <td>
+                      <a href={config.apiUrl + "/packages/" + dependent.id}>{dependent.id}</a>
```

Comment:
Think can be:

```suggestion
                      <Link to {`/packages/${dependent.id}`}>{dependent.id}</Link>
```

For example, see how search results link to the package page: https://github.com/loic-sharma/BaGet/blob/7c514bfe57a59b7ed086e37b73d61a8194306a20/src/BaGet.UI/src/SearchResults.tsx#L198